woodwork/datatable.py

```diff
@@ -563,7 +571,12 @@ def remove_semantic_tags(self, semantic_tags):
             woodwork.DataTable: DataTable with the specified semantic tags removed
         """
         _check_semantic_tags(self._dataframe, semantic_tags)
-        return self._update_cols_and_get_new_dt('remove_semantic_tags', semantic_tags)
+        dt = self._update_cols_and_get_new_dt('remove_semantic_tags', semantic_tags)
+        for _, tags in semantic_tags.items():
```

Comment:
Can just do `for tags in semantic_tags.values()` here since we discarding the keys.